src/library_exceptions.js

```diff
@@ -337,9 +337,15 @@ var LibraryExceptions = {
     // Remove this JS function name, which is in the second line, from the stack
     // trace. Note that .stack does not yet exist in all browsers (see #18828).
     if (e.stack) {
-      var arr = e.stack.split('\n');
-      arr.splice(1,1);
-      e.stack = arr.join('\n');
+      // WebAssembly.Exception.stack is currently readonly, but this can change,
+      // and it is implemented as writable in Chrome and Node. Will check the
+      // 'writable' condition for now and later resolve this by either deleting
+      // this altogether or removing this condition check.
+      if (Object.getOwnPropertyDescriptor(e, 'stack').writable) {
```

Comment:
Combine this with the above into a single `if` ?

Did you verify that this is indeed readonly on FF for you locally?

Reply:
Combined within a single `if`. No I haven't verified that locally because I don't have FF installed... I'll add a browser test so it will be confirmed by the CI.

Reply:
I would recommend confirming the issue locally, just to be sure.   Is there some reason you don't want to install FF?